zerver/lib/bugdown/fenced_code.py

```diff
@@ -252,6 +264,37 @@ def done(self) -> None:
         self.output.append('')
         self.processor.pop()
 
+
+class ZpoilerHandler(BaseHandler):
```

Comment:
I assume this is implied by my comments on naming above, but this should be just `SpoilerHandler` (etc.).